R/mod_tableGen.R

```diff
@@ -497,8 +497,8 @@ mod_tableGen_server <- function(input, output, session, datafile = reactive(NULL
       mutate(
         ID = purrr::reduce(
           list(
-            pattern = stringr::str_c('\\b', pretty_blocks$Pattern, '\\b', sep = ''),
-            replacement = pretty_blocks$Replacement
+            pattern = stringr::str_c('\\b', tidyCDISC::pretty_blocks$Pattern, '\\b', sep = ''),
+            replacement = tidyCDISC::pretty_blocks$Replacement
```

Comment:
To me part of the problem is that we are having our caking and wanting to eat it too. We want to use `pretty_blocks` within our coding but to also provide the file to the end user, which is [against convention](https://cran.r-project.org/doc/manuals/R-exts.html#Data-in-packages).

Reply:
`pretty_IDs(0` is a great solution. Love it! But what about our CDISC pilot data? We export & use that data internally. Do we just put copies of that data into `inst/extdata` according to convention? That's seems... redundant/unnecessary.

Reply:
@Jeff-Thompson12, here!

Reply:
> @Jeff-Thompson12, ah, so using `tidyCDISC::adsl`, etc? I'm cool with that. Thanks for sending the article - it was helpful for my understanding.

@AARON-CLARK I have made the changes requested for this branch. Dealing with pulling in the data like `tidyCDISC::adsl` will be a little more complicated. Currently the code is running `eval(parse(text = "adsl"))` because we need to pull the test data in as a list. I was going to tackle that problem in a different PR though.

Reply:
@Jeff-Thompson12 Okay, I will leave this conversation as "unresolved" but still merge it with `for_cranny`.